perfetto-sys/cpp/wrapper.h

```diff
@@ -11,7 +11,13 @@ enum class ArgType : uint8_t {
     I64KeyValue,
     U64KeyValue,
     BoolKeyValue,
-}; 
+};
+
+/// Event types for the PerfettoEventArg struct.
+enum class EventType {
+	Begin,
```

Comment:
Maybe `Span`? `Begin` is not an event type, but rather specifies that this is the beginning of the span